diff -u php8.1.conf.orig php8.1.conf
--- /etc/apache2/mods-available/php8.1.conf	2023-07-19 11:34:30.478398010 -0600
+++ /etc/apache2/mods-available/php8.1.conf	2023-07-19 11:34:52.224008006 -0600
@@ -12,14 +12,3 @@
 <FilesMatch "^\.ph(ar|p|ps|tml)$">
     Require all denied
 </FilesMatch>
-
-# Running PHP scripts in user directories is disabled by default
-# 
-# To re-enable PHP in user directories comment the following lines
-# (from <IfModule ...> to </IfModule>.) Do NOT set it to On as it
-# prevents .htaccess files from disabling it.
-<IfModule mod_userdir.c>
-    <Directory /home/*/public_html>
-        php_admin_flag engine Off
-    </Directory>
-</IfModule>
diff -u userdir.conf.orig userdir.conf
--- /etc/apache2/mods-available/userdir.conf	2023-07-19 11:35:12.148053001 -0600
+++ /etc/apache2/mods-available/userdir.conf	2023-07-19 11:35:30.370053010 -0600
@@ -3,7 +3,7 @@
 	UserDir disabled root

 	<Directory /home/*/public_html>
-		AllowOverride FileInfo AuthConfig Limit Indexes
+		AllowOverride FileInfo AuthConfig Limit Indexes Options
 		Options MultiViews Indexes SymLinksIfOwnerMatch IncludesNoExec
 		Require method GET POST OPTIONS
 	</Directory>
