diff -u -r php7.4.conf php7.4.conf
--- /etc/apache2/mods-available/php7.4.conf	2020-01-21 15:37:04.749186590 -0800
+++ /etc/apache2/mods-available/php7.4.conf	2020-01-21 15:40:25.181194660 -0800
@@ -12,14 +12,3 @@
 <FilesMatch "^\.ph(ar|p|ps|tml)$">
     Require all denied
 </FilesMatch>
-
-# Running PHP scripts in user directories is disabled by default
-# 
-# To re-enable PHP in user directories comment the following lines
-# (from <IfModule ...> to </IfModule>.) Do NOT set it to On as it
-# prevents .htaccess files from disabling it.
-<IfModule mod_userdir.c>
-    <Directory /home/*/public_html>
-        php_admin_flag engine Off
-    </Directory>
-</IfModule>
diff -u -r userdir.conf userdir.conf
--- /etc/apache2/mods-available/userdir.conf	2020-03-18 19:38:22.270154775 -0600
+++ /etc/apache2/mods-available/userdir.conf	2020-03-18 19:38:13.762410778 -0600
@@ -3,7 +3,7 @@
 	UserDir disabled root

 	<Directory /home/*/public_html>
-		AllowOverride FileInfo AuthConfig Limit Indexes
+		AllowOverride FileInfo AuthConfig Limit Indexes Options
 		Options MultiViews Indexes SymLinksIfOwnerMatch IncludesNoExec
 		Require method GET POST OPTIONS
 	</Directory>
