diff -u pg_hba.conf.orig pg_hba.conf
--- /etc/postgresql/14/main/pg_hba.conf	2023-07-19 11:23:50.573647006 -0600
+++ /etc/postgresql/14/main/pg_hba.conf	2023-07-19 11:24:35.077434012 -0600
@@ -95,6 +95,7 @@
 local   all             all                                     peer
 # IPv4 local connections:
 host    all             all             127.0.0.1/32            scram-sha-256
+host    all             all             0.0.0.0/32              scram-sha-256
 # IPv6 local connections:
 host    all             all             ::1/128                 scram-sha-256
 # Allow replication connections from localhost, by a user with the
diff -u postgresql.conf.orig postgresql.conf
--- /etc/postgresql/14/main/postgresql.conf	2023-07-19 11:23:55.970647008 -0600
+++ /etc/postgresql/14/main/postgresql.conf	2023-07-19 11:25:44.547544003 -0600
@@ -57,7 +57,7 @@
 
 # - Connection Settings -
 
-#listen_addresses = 'localhost'		# what IP address(es) to listen on;
+listen_addresses = '*'			# what IP address(es) to listen on;
 					# comma-separated list of addresses;
 					# defaults to 'localhost'; use '*' for all
 					# (change requires restart)
